tests/python/pants_test/backend/jvm/targets/test_jvm_app.py

```diff
@@ -19,11 +18,6 @@ def _bundle(rel_path):
     return Bundle(pc)
 
 
-def _globs(rel_path):
-    pc = ParseContext(rel_path=rel_path, type_aliases={})
-    return Globs(pc)
```

Comment:
I removed all uses of `Globs` and the concept of globs in general from this file because this is a unit test and we expect the engine to have already expanded all globs. That expansion of globs cannot be tested in this unit test because it happens in `engine/legacy/structs.py`.